docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+# For more information: https://laravel.com/docs/sail
+version: '3'
+services:
+  app:
+    container_name: ${DOCKER_CONTAINER_NAME:-protector_test}
+    build:
+      context: ./docker/${DOCKER_PHP_VERSION:-8.1}
+      dockerfile: Dockerfile
+      args:
+        WWWGROUP: '${WWWGROUP}'
+    image: sail-${DOCKER_PHP_VERSION:-8.1}/app
+    extra_hosts:
+      - 'host.docker.internal:host-gateway'
+    environment:
+      WWWUSER: '${WWWUSER}'
+      LARAVEL_SAIL: 1
+      XDEBUG_MODE: '${SAIL_XDEBUG_MODE:-off}'
+      XDEBUG_CONFIG: '${SAIL_XDEBUG_CONFIG:-client_host=host.docker.internal}'
+    volumes:
+      - '.:/var/www/html'
+    networks:
+      - shared
+      - internal
+    depends_on:
+      - mysql_testing
+  mysql_testing:
+    image: 'mysql/mysql-server:8.0'
+    environment:
+      MYSQL_ROOT_PASSWORD: '${DB_PASSWORD}'
+      MYSQL_ROOT_HOST: "%"
+      MYSQL_DATABASE: '${DB_DATABASE:-protector_test}'
+      MYSQL_USER: '${DB_USERNAME}'
+      MYSQL_PASSWORD: '${DB_PASSWORD}'
+      MYSQL_ALLOW_EMPTY_PASSWORD: 1
+    networks:
+      - internal
+    healthcheck:
+      test: ["CMD", "mysqladmin", "ping", "-p${DB_PASSWORD}"]
+      retries: 3
+      timeout: 5s
+networks:
+  internal:
+    internal: true
+  shared:
+    external: true
+volumes:
+  sail-mysql:
+    driver: local
```

Comment:
Why would you want a mounted volume to persist data?

Reply:
no reason